docs/en/integrations/data-ingestion/data-formats/binary.md

```diff
@@ -0,0 +1,220 @@
+---
+sidebar_label: Binary and native
+sidebar_position: 5
+slug: /en/integrations/data-formats/binary-native
+---
+
+# Using native and binary formats in ClickHouse
+
+ClickHouse supports multiple binary formats, which result in better performance and space efficiency. Binary formats are also safe in character encoding since data is saved in a binary form.
+
```

Comment:
should we populate `some_data` here?

Reply:
Sounds reasonable, I added SQL/TSV to the assets folder and a sentence to reference it.